### ISSUE TITLE
Question for editor

### ISSUE THREAD

**Issue by Tigerma0707** [frozenwolf64/Adapt-QAOA#2]
In your (derivative(hamil, mixer, qc)), I can get the different return-values every run, why?  What this function do is getting the expectation of commutator ?

**Comment by frozenwolf64** [comment]
Yes, since the function is defined with finitely many shots (1024) the expectation value will be slightly different each run due to the probabilistic nature of quantum circuits. It should become more consistent as you take more shots.
